src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryDeploymentProperties.java

```diff
@@ -186,11 +186,6 @@
 	 */
 	private boolean autoDeleteMavenArtifacts = true;
 
-	/**
-	 * A map containing environment variables.
-	 */
-	private Map<String, String> env = Collections.emptyMap();
```

Comment:
This is part of an api so we can't just remove it.

Reply:
Good catch. Restored, but prevent from being changed once statically configured